test/v200/apiTest/api_test.go

```diff
@@ -0,0 +1,80 @@
+package apiTest
+
+import (
+	"testing"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
```

Comment:
same comment as the other pkg alias, consider renaming or not at all

Reply:
I plan in my next pull request to make these commonUtils and apiUtils.

Reply:
Fixed in new commit